src/Utopia/Messaging/Adapter/SMS/Mock.php

```diff
@@ -69,4 +72,27 @@ protected function process(SMSMessage $message): array
 
         return $response->toArray();
     }
+
+    /**
+     * @param string $phone
+     * @return int|null
+     * @throws Exception
+     */
+    public function getCountryCode(string $phone): ?int
```

Comment:
why have we defined this again in the Mock adapter? Isnt it already present in Adapter.php?